pennylane/tape/qscript.py

```diff
@@ -91,6 +91,8 @@ class QuantumScript:
         prep (Iterable[Operator]): Any state preparations to perform at the start of the circuit
 
     Keyword Args:
+        shots (None, int, Sequence[int], ~.Shots): Number and/or batches of
+            shots for execution
```

Comment:
```suggestion
        shots (None, int, Sequence[int], ~.Shots): Number and/or batches of
            shots for execution. Note that this property is still experimental and under development.
```

For the time being, best to clarify that even though shots can be specified on the tape, that might not actually do anything.